react/AddToCartButton.tsx

```diff
@@ -133,6 +137,10 @@ function AddToCartButton(props: Props) {
     event.stopPropagation()
     event.preventDefault()
 
+    if (goToProductPage && linkText) {
+      return navigate({ to: `/${linkText}/p` })
```

Comment:
Instead of creating the link you can just use the options [like this](https://github.com/vtex-apps/product-summary/blob/master/react/components/ProductSummary.js#L121-L128). These props are being passed to a `Link`, but as far as a remember the props are the same

Reply:
Thank you @Klynger 
I did this change

Reply:
Thx, after you make the changes you can request my review again, if you do it I can receive a notification in slack, so it's easier to see